docs/whats-new.md

```diff
@@ -1,5 +1,19 @@
 # What's new
 
+## react-map-gl v8.0
+
+Release date: Jan 2025
+
+- First version to support Mapbox GL JS' official types and MapLibre GL JS v5.
+  This version fully separates the code that support each compatible map library:
+  + `react-map-gl/mapbox`: for use with `mapbox-gl>=3.5.0`
+  + `react-map-gl/maplibre`: for use with `maplibre-gl>=4`
+  + `react-map-gl/mapbox-legacy`: for use with `mapbox-gl` v1.x and v2.x and `@types/mapbox-gl`
```

Comment:
What will the old endpoint "react-map-gl" point to? If it's removed, should we note what to change it to?

Reply:
`react-map-gl` no longer has any default export. This is noted in the upgrade guide.